antareslauncher/study_dto.py

```diff
@@ -43,7 +43,7 @@ class StudyDTO:
     # Simulation stage data
     time_limit: t.Optional[int] = None
     n_cpu: int = 1
-    antares_version: int = 0
+    antares_version: StudyVersion = StudyVersion.parse(0)
```

Comment:
I don't think the parsing from tiny db will work ?
Also we need to take care that reading "old style" DTOs from tinydb still works, otherwise we will get exceptions when transitioning to this new version with old DTOs on disk

Reply:
The parsing is done inside method `save_study` and I wrote this line of code which I'm not super proud of: 
`new["antares_version"] = f"{new['antares_version']:2d}"`  line 93 of the file data_repo_tinydb.py

Reply:
Thanks, I still added a smalle unit test to test this.